shell/src/main/java/org/apache/zeppelin/shell/ShellInterpreter.java

```diff
@@ -93,18 +95,31 @@ public InterpreterResult interpret(String cmd, InterpreterContext contextInterpr
         + " return with exit value: " + exitVal);
       return new InterpreterResult(Code.SUCCESS, outStream.toString());
     } catch (ExecuteException e) {
-      int exitValue = e.getExitValue();
-      LOGGER.error("Can not run " + cmd, e);
-      Code code = Code.ERROR;
-      String message = outStream.toString();
-      if (exitValue == 143) {
-        code = Code.INCOMPLETE;
-        message += "Paragraph received a SIGTERM\n";
-        LOGGER.info("The paragraph " + contextInterpreter.getParagraphId() 
-          + " stopped executing: " + message);
+      if (isGSSInit &&
+          contextInterpreter.out != null &&
+          contextInterpreter.out.getCurrentOutput().toString().contains("GSS")) {
```

Comment:
Do you mean to replace `GSS` with `GSSException` or even longer like `Caused by GSSException: No valid credentials provided`?